### ISSUE TITLE
Auto front light plugin improvement

### ISSUE THREAD

**Issue by Hzj-jie** [koreader/koreader#2963]
* KOReader version: 1058
* Device: Kindle Voyage

#### Issue

1. A nertual range between bright and dim. When the environment brightness is in this range, the state of front light should not be changed.
2. Footer is not updated. The implementation in Kobo light is too hacky. We should add an event.
3. Update frequency can be lower.
4. Force mode? I.e. it should not override users decision. If user turned on or off the light, the plugin should stop working for a while, or until a significant change of the brightness.
5. Configuration to turn on / off the feature and some comments.

**Comment by Frenzie** [comment]
> But why it adresses Kobo light then?

Maybe it was used as the basis? There has been at least some refactoring of the brightness/warmth stuff since but at one point something in that general area was all copied from Kobo repeatedly.

**Comment by NiLuJe** [comment]
Yeah, that was mostly aimed at the Voyage's ALS. We've never implemented ALS anywhere else since (for... good reasons. ALS are a terrible idea on eInk ;D).

**Comment by Frenzie** [comment]
Yeah, it sounds like a really weird idea. It should work more or less the reverse of regular LCD/OLED behavior but even then that's a weird and unnecessary delay.

**Comment by artjomsR** [comment]
Chiming in to say that I'd like to see 
> 4. Force mode? I.e. it should not override users decision. If user turned on or off the light, the plugin should stop working for a while, or until a significant change of the brightness.
as well

I think the settings should be maintained until the sun cycle changes. Motivation is that I've configured the light settings that work for me in general. But in an occasionally different environment (I'm sitting somewhere too bright / dark) I need a way to override the defaults during a reading session. Basically, if the sun cycle is the same, and I've changed books, there's no reason to revert brightness levels

I think https://github.com/koreader/koreader/issues/9728 is a duplicate, but that was fixed + closed somehow, although the issue doesn't seem to be resolved?

**Comment by pazos** [comment]
Closed by #12809
